api/android/api/src/main/java/org/nnsuite/nnstreamer/SingleShot.java

```diff
@@ -54,7 +54,7 @@
      * @throws IllegalStateException if this failed to construct the pipeline
      */
     public SingleShot(@NonNull File model) {
-        this(model, null, null);
+        this(new File[]{model}, null, null, NNStreamer.NNFWType.TENSORFLOW_LITE, null);
```

Comment:
Why TFLITE is chosen here?

Reply:
constructor `public SingleShot(@NonNull File model)` with default tf-lite now.